File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.cc
#include<iostream>
using namespace std;

int arr[31];

int main()
{
	for (int i = 0; i < 28; i++) {
		int a; cin >> a;
		arr[a] = 1;
	}

	for (int i = 1; i <= 30; i++) {
		if (!arr[i]) cout << i << '\n';
	}
	
	return 0;
}

File: 백준/Bronze/5597. 과제 안 내신 분．．？/과제 안 내신 분．．？.java
import java.io.*;
import java.util.*;

class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		int[] arr = new int[35];
		for(int i = 0; i < 28; i++){
			int n = Integer.parseInt(br.readLine());
			arr[n-1] = 1;
		}
		
		for(int i = 0; i < 30; i++){
			if(arr[i] == 0) System.out.println(i+1);
		}
	}
}